generator/src/main/resources/line-bot-sdk-ruby-rbs-generator/api.pebble

```diff
@@ -9,6 +9,33 @@ module Line
       module {{ packageName | camelize }}
         class Api{{ operations.classname.indexOf('Blob') != -1 ? 'Blob' : '' }}Client
           @http_client: HttpClient
+          # Initializes a new {Line::Bot::V2::{{ packageName | camelize }}::Api{{ operations.classname.indexOf('Blob') != -1 ? 'Blob' : '' }}Client} instance.
+          #
+          # @param base_url [String] The base URL for requests (optional).
+          #   Defaults to '{{endpoint(operations.classname)}}' if none is provided.
```

Comment:
If we want to write the exact same comment, it's probably correct to extract it into a separate pebble file. This change doesn't create another pebble file(but it should be done ideally).